src/linked_list.py

```diff
@@ -0,0 +1,59 @@
+class Node:
+    def __init__(self, val, next_node):
+        self.val = val
+        self.next_node = next_node
+
+    def __repr__(self):
+        return f"({self.val})->{self.next_node}"
+
+
+class LinkedList:
+    def __init__(self, iterable=None):
+        self.head = None
+        self._length = 0
+        if type(iterable) in [list, tuple, str]:
```

Comment:
Typically `isinstance()` is preferred in Python over `type`, as it works with more use case. It also let's you test if "some variable is ONE of a tuple of possible types". EX:

```python
my_variable = 1
if isinstance(my_variable, (str, int)):
    print("yay!")
```

Reference: https://towardsdatascience.com/difference-between-type-and-isinstance-in-python-47fae6fbb068